src/python/deepgnn/graph_engine/snark/synchronized2.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
NOTE This is old code for backup

Reply:
Please, remove old code, so it is visible in PR.